src/main/java/com/airbnb/billow/AWSDatabase.java

```diff
@@ -79,6 +77,37 @@
         this.ec2SGs = ec2SGbuilder.build();
 
         /*
+         * IAM keys
+         */
+
+        log.info("Getting IAM keys");
+        String accountNumber = ""; // we need this to describe RDS
+        final ImmutableList.Builder<IAMUserWithKeys> usersBuilder = new ImmutableList.Builder<IAMUserWithKeys>();
+
+        final ListUsersRequest listUsersRequest = new ListUsersRequest();
+        ListUsersResult listUsersResult;
+        do {
+            log.debug("Performing IAM request: {}", listUsersRequest);
+            listUsersResult = iamClient.listUsers(listUsersRequest);
+            final List<User> users = listUsersResult.getUsers();
+            log.debug("Found {} users", users.size());
+            for (User user : users) {
```

Comment:
that seems somewhat reasonable although kind of confusing. Can we make it an optional argument that attempts to fetch from args and falls tback to pulling from this?